index.html

```diff
@@ -1,92 +1,111 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
+<head>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Quicksand:wght@300;500&display=swap" rel="stylesheet">
     <meta charset="UTF-8">
-    <title>Fork Me! FCC: Test Suite Template</title>
-    <link rel="stylesheet" href="css/style.css">
-	  <style>
-		  img {
-			  max-width: 100%;
-			  display: block;
-			  height: auto;
-			  margin: 0 auto;
-		  }
-		  #img-div {
-
-			  padding: 10px;
-			  margin: 0;
-		  }
-
-		  #main {
-			  margin: 30px 8px;
-			  padding: 15px;
-			  border-radius: 5px;
-			  background: #eee;
-		  }
-
-
-
-
-	  </style>
```

Comment:
Can you plese remove inline css that you used in this file create a different file for it it should be better